src/lib/ChargeStation/eventHandlers/ocpp-20/send-authorize.ts

```diff
@@ -11,7 +11,7 @@ const sendAuthorize: ChargeStationEventHandler<
 
   chargepoint.writeCall(
     'Authorize',
-    { idToken: { idToken: session.options.uid, type: 'ISO14443' } },
+    { idToken: { idToken: session.options.uid, type: session.options.idTokenType || session.options.authorizationType  } },
```

Comment:
This makes sense, and will work, but I generally feel it would be nice to keep anything protocol specific out of the `authorizationType` value. Or put differently, ideally the code shouldn't assume that `authorizationType` holds a valid value for OCPP 2.0.1 (which it didn't before the enum was changed). My reasoning is that the `authorizationType` field is used by the OCPP 1.6 handlers too, and whilst those bits of code don't care for the underlying value, it just feels slightly quirky to have this aligned to a specific version of the protocol. So probably `session.options.idTokenType || 'ISO14443'` would be fine, or a 2.0.1 specific function could be added if there is a need for anything more elaborate.